### ISSUE TITLE
Expose standard authenticator fields in JS objects, vs. binary

### ISSUE THREAD

**Issue by bifurcation** [w3c/webauthn#337]
The result of a `makeCredential` call provides certain information about a credential:

* Type
* ID
* Public key
* RP ID associated to the credential

Of these four elements, only the Type and ID are exposed in the `ScopedCredential` object.   The other two are buried in the `authenticatorData` field of the `AuthenticatorAttestation` object.  For the RP ID, that's fine; this is just reflecting back a value the caller specified.  The public key, however, is generated by the the authenticator, and needs to be accessible to the RP in order for it to verify assertions.  

Instead of forcing the RP to parse the public key out of the `authenticatorData`, the API should expose the public key as an attribute on `ScopedCredential` in a way that can be more directly used, e.g., as a JWK that can be used with WebCrypto.

**Comment by bifurcation** [comment]
@vijaybh tells me that this is already sort of handled in #321, in that all of the above fields are presented in a standard format, regardless of the attetastion type.  That addresses the main concern, that RPs that don't care about attestion not have to deal with attestation-specific things.  

I would still prefer that these things be exposed in a more JS-friendly format, so that the attestation-independent parts can easily be done in JS.  @balfanz and I will come up with a proposal once #321 lands.

**Comment by selfissued** [comment]
https://github.com/w3c/webauthn/pull/321 has landed.  We need to finish deciding this for the Implementer's Draft.  (It could be that we decide that there's nothing left to do.)

**Comment by selfissued** [comment]
There was agreement on the call that this unnecessary.  We should close it as won't fix.

**Comment by equalsJeffH** [comment]
the agreement at the Feb webauthn F2F was that @bifurcation and @balfanz would come up with a proposal in a few weeks, or we would close this. No proposal has emerged, plus as noted above, the webauthn spec as of merge of pr #321 features an attestation data object containing the credential in a consistent format regardless of the attestation statement format in use. See https://w3c.github.io/webauthn/#fig-attStructs

So closing this issue wontfix
